chromium/background.js

```diff
@@ -573,7 +573,7 @@ function onBeforeRedirect(details) {
 
 // Registers the handler for requests
 // See: https://github.com/EFForg/https-everywhere/issues/10039
-wr.onBeforeRequest.addListener(onBeforeRequest, {urls: ["<all_urls>"]}, ["blocking"]);
+wr.onBeforeRequest.addListener(onBeforeRequest, {urls: ["*://*/*"]}, ["blocking"]);
```

Comment:
This was changed to `<all_urls>` due to reasoning I describe in https://github.com/EFForg/https-everywhere/issues/10039#issuecomment-307276140.  If this changes, we'll have to test this in `webextensions` as well.